common/config/rush/browser-approved-packages.json

```diff
@@ -1,5 +1,30 @@
 // DO NOT ADD COMMENTS IN THIS FILE.  They will be lost when the Rush tool resaves it.
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
-  "packages": []
+  "packages": [
+    {
+      "name": "@jest/core",
+      "allowedCategories": [ "libraries" ]
```

Comment:
@sachinjoseph Please move these to `nonbrowser-approved-packages.json` since they are used by build tools, not a web app.